examples/pytorch/tagcn/README.md

```diff
@@ -0,0 +1,24 @@
+Topology Adaptive Graph Convolutional networks (TAGCN)
+============
+
+- Paper link: [https://arxiv.org/abs/1710.10370](https://arxiv.org/abs/1710.10370)
+
+Dependencies
+------------
+- PyTorch 0.4.1+
+- requests
+
+``bash
+pip install torch requests
+``
+
+Results
+-------
+Run with following (available dataset: "cora", "citeseer", "pubmed")
+```bash
+python3 train.py --dataset cora --gpu 0 --self-loop
+```
+
+* cora: ~0.812 (0.804-0.823) (paper: 0.833)
+* citeseer: ~0.715 (paper: 0.714)
+* pubmed: ～0.790 (paper: 0.811)
```

Comment:
Try tuning some hyper-parameters like weight_decay and see if the result is still lower then the number in TAGCN paper.

Reply:
I tried weight_decay and lr for pubmed dataset, there is minor performance improvement.
The new test result was updated.